### ISSUE TITLE
weather temperature reports incorrect Fahrenheit value 

### ISSUE THREAD

**Issue by grewhit25** [briis/smartweather#30]
HACS config:

```yaml
Authors: briis
GitHub stars: 23
Last updated: 2 days ago
Installed: V1.0.2
```

My weather component is still showing the incorrect temperature similar to #28 but for weather.
All other weather forecast values appears correctly in Fahrenheit except for the temperature which is already in Fahrenheit but got converted as if it's being interpreted as Celsius.

My weather sensor configured is:

```yaml
weather:
  - platform: smartweather
    api_key: !secret darksky_key
```

**Comment by briis** [comment]
Ok. Thanks for reporting this. I will look in to it.

**Comment by briis** [comment]
Fixed in release V1.0.3, which has just been published.
